migration/sql-test-files/053-edit-username.sql

```diff
@@ -0,0 +1,22 @@
+-- users
+INSERT INTO
+   users(created_at, updated_at, id, email, full_name, image_url, bio, url, context_information)
+VALUES
+   (
+      now(), now(), 'f03f023b-0427-4cdb-924b-fb2369018aa7', 'testtwo@example.com', 'test one', 'https://www.gravatar.com/avatar/testtwo', 'my test bio one', 'http://example.com', '{"key": "value"}'
```

Comment:
users ids are also conflicting with the existing ones, added in the insert-test-data.sql script